oecologia-australis.csl

```diff
@@ -61,9 +61,9 @@
     <choose>
       <if type="article-journal chapter paper-conference entry-dictionary entry-encyclopedia" match="none">
         <group delimiter=", ">
-          <names variable="editor translator" delimiter=", " suffix=" ">
```

Comment:
So the new guide you posted still lists translators for books, albeit in consistently:
Borges, J. L. 1999. Selected non-fictions. E. Weinberger (Ed.), **E. Allen, S. J. Levine, & E. Weinberger, Trans.** New York: Viking: p. 559.

I think it would be odd to leave translators out of a citations, but I agree with the change of the label, let's change line 64 back to "editor translator" but keep the rest of the changes

Reply:
Translator was excluded by new rules. As I said you before, this rules found in link are not updated, also by this I as reluctant in add this link.